### DISCUSSION TITLE
请问在summary sheet 页面中如何去掉标题下的summary，谢谢了

### DISCUSSION THREAD

**Discussion by R-NoXX** [xjtu-blacksmith/easymcm#4] [Q&A] [answered]
![V$({5(_(RS8 3IQ`6T8MS V](https://user-images.githubusercontent.com/77869503/105567773-8475d580-5d6f-11eb-9444-5e779e7afa3a.png)

**Comment by xjtu-blacksmith** [comment, marked as answer]
通过清空摘要标题的宏 `\abstractname` 来解决。请在导言区中（载入 `easymcm` 宏包的命令后）新增以下命令：

```latex
\renewcommand\abstractname{}
```

即可清空 Summary 字样，如下图：

![图片](https://user-images.githubusercontent.com/42454373/105620781-5c53a880-5e3b-11eb-9519-0b5ae3aea0f5.png)
